apm-lambda-extension/extension/apm_server.go

```diff
@@ -24,14 +24,32 @@ import (
 	"io"
 	"io/ioutil"
 	"log"
+	"math"
+	"math/rand"
 	"net/http"
 	"sync"
+	"time"
 )
 
 var bufferPool = sync.Pool{New: func() interface{} {
 	return &bytes.Buffer{}
 }}
 
+type APMServerTransportStatusType string
+
+const (
+	failure APMServerTransportStatusType = "failing"
+	healthy APMServerTransportStatusType = "healthy"
+)
+
+var apmServerTransportStatus APMServerTransportStatusType
```

Comment:
As it can only have two states, what about just using a bool?

Reply:
Solved in ad48058